41_advanced_node_management/20_extend_node_inventory.txt

```diff
@@ -0,0 +1,100 @@
+[[extend-nodes-inventory, Extend node inventory]]
+
+=== Extend node inventory
+
+
+It is quite common to need to gather information on your nodes that are not present
+in the standard Rudder inventory information. 
+
+As of Rudder 4.3.0, you can add get more information about a node thanks to 
```

Comment:
s/add//

Reply:
I don't understand, I removed it before commiting... Well, perhaps the day should end quickly :)

Reply:
aaaaahhhh. That editor need to be hit "ctrl+s" to get the change :)